savant/app/results.py

```diff
@@ -25,11 +27,26 @@ def compare():
     get_scanned.get_items(filenames)
     record.record_comparison_flavors()
 
+    json_file = "Comparison-" + now.strftime("%Y-%m-%d_%H:%M") + ".json"
+
+    new_packages =  json.dumps({"Debs" : {"New" : comparison.new_debs()}},
```

Comment:
This failed the Code Climate test; can you refactor this as a single function? See https://codeclimate.com/github/Doveps/mono/pull/51